Tests/TestsCommon/SpecTestUtils.swift

```diff
@@ -1,5 +1,5 @@
 import Foundation
-@testable import MongoSwift
+import MongoSwift
```

Comment:
I moved this file into common because some of these utils are used by the BSON corpus tests.